src/mpid/ch4/include/mpidpre.h

```diff
@@ -23,6 +23,8 @@
 #include "uthash.h"
 #include "ch4_csel_container.h"
 
+#define MPID_TAG_DEV_BITS 0
```

Comment:
is this correct? `ch4/ofi` utilizes tag bits for things link synchronous and huge send protocols to my recollection?

Reply:
I think where I am confused is when `ch4/ofi` limits the user tag bits to 20 (`MPIDI_OFI_TAG_BITS_a`), and `MPIR_Process.tag_bits` get set to reflect that, but this value never changes.

Reply:
They are used in `ofi_capability_sets.h`. Whether it's `_a` or `_b` depends on whether the provider supports CQ data.

Reply:
Long story short this define is unused so is harmless. A future PR may use it and there we can sort out how it should be used in conjunction with `MPIR_Process.tag_bits`.